themes/gohugoioTheme/layouts/_default/baseof.html

```diff
@@ -7,6 +7,8 @@
     <link rel="preload" href="{{ "files/muli-latin-400.woff2" | absURL }}" as="font" type="font/woff2" crossorigin>
     <link rel="preload" href="{{ "files/muli-latin-800.woff2" | absURL }}" as="font" type="font/woff2" crossorigin>
 
+    <link href="/news/index.xml" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
```

Comment:
Strictly speaking, it is not entirely correct to use the site's title for this. And this is a general theme used by several sites, including the theme site. 

I suggest that we:

* Use `GetPage` to get the news section it exists
* Use `.OutputFormats` to get the `RSS` link / title